client/src/components/reader/Header.js

```diff
@@ -157,7 +159,21 @@ export default class Header extends Component {
           visible={this.props.visibility.tocDrawer}
           hideTocDrawer={this.triggerHideToc}
         />
-        <nav className="menu-panels">
+        <nav className="menu-panels-left">
+          <UIPanel
+            id="readerReturn"
+            visibility={this.props.visibility.uiPanels}
+            bodyComponent={ReturnMenuBody}
+
+            // Props required by body component
+            projectId={this.props.text.relationships.project.id}
+            projectTitle="The Project Title"
+            toggleSignInUpOverlay={this.props.commonActions.toggleSignInUpOverlay}
+            moreLink="http://manifold.umn.edu/about/"
```

Comment:
Please add a `TODO` comment so we have a note that this needs to be pulled from settings. I know there's a story, but good to track this sort of TODO in code as well.